.studio/automate-gateway

```diff
@@ -138,8 +138,14 @@ function send_chef_action_example() {
   rfc_time=$(date --rfc-3339=seconds  -d "$((RANDOM % 144)) hour ago" | sed 's/ /T/' | sed 's/\+.*/Z/')
 
   # Update the 'recorded_at' time from the ChefAction Example; and set entity_type and task
-  tmp_action_json="$(jq --arg event_task "$event_task" --arg rfc_time "$rfc_time" --arg event_type "$event_type" \
-    '.task = $event_task | .recorded_at = $rfc_time | .entity_type = $event_type' < $examples_path/chef_action.json)"
+  tmp_action_json="$(jq --arg event_task "$event_task" \
+                        --arg rfc_time "$rfc_time" \
+                        --arg event_type "$event_type" \
+                        --arg random_key "$(head -c 100 /dev/urandom | tr -cd 'a-zA-Z')" \
```

Comment:
I think I'd prefer you generate some nonsense just once and then apply it statically here. If this fails it might be difficult to reproduce if the value is not logged (or if it's hard to find the logs or the log entry). Fuzzy type testing is cool but I think you really need the additional tooling for showing and manipulating the RNG seed to get good value.